sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsOptions.cs

```diff
@@ -47,14 +47,15 @@ public int MaxDegreeOfParallelism
         }
 
         /// <summary>
-        /// Gets or sets the number of times to try processing a message before moving it to the poison queue (where
-        /// possible).
+        /// Gets or sets the number of times to try processing a given blob before adding a message to a
+        /// storage queue named, webjobs-blobtrigger-poison.
```

Comment:
nit (errant comma). Suggest "Gets or sets the number of times to try processing a blob before giving up and creating a poison queue message for it in queue 'webjobs-blobtrigger-poison'."

Reply:
@mathewc is the update I made good?